dev-qt/qtbase/qtbase-6.7.2-r1.ebuild

```diff
@@ -56,6 +56,7 @@ RDEPEND="
 	ssl? ( dev-libs/openssl:= )
 	udev? ( virtual/libudev:= )
 	zstd? ( app-arch/zstd:= )
+	systemd? ( sys-apps/systemd )
```

Comment:
should be sorted alphabetically in the "flag?" group

Reply:
Done